firefox-ios/firefox-ios-tests/Tests/ClientTests/Frontend/Homepage Rebuild/LabelButtonHeaderViewTests.swift

```diff
@@ -0,0 +1,40 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+import Common
+
+@testable import Client
+
+final class LabelButtonHeaderViewTests: XCTestCase {
+    var view: LabelButtonHeaderView!
```

Comment:
I think here instead of having a property for the view, we could as like the other tests having a method the creates the view like. So we can avoid having an implicit unwrapped optionals and to have to keep track of class state.
```swift
private func createSubject() -> LabelButtonHeaderView {
}
```